api/registrations/views.py

```diff
@@ -850,3 +853,44 @@ def get_default_queryset(self):
 
     def get_queryset(self):
         return self.get_queryset_from_request()
+
+
+class RegistrationSchemaResponsesList(JSONAPIBaseView, ListFilterMixin, generics.ListCreateAPIView, RegistrationMixin):
+    required_read_scopes = [CoreScopes.NULL]
+    required_write_scopes = [CoreScopes.NULL]
+
+    permission_classes = (
+        drf_permissions.IsAuthenticated,
+        base_permissions.TokenHasScope,
+        AdminContributorOrPublic,
+    )
+
+    view_category = 'registrations'
+    view_name = 'schema-responses-list'
+
+    serializer_class = SchemaResponsesListSerializer
+
+    def get_default_queryset(self):
+        return self.get_node().schema_responses.all()
+
+    def get_queryset(self):
+        return self.get_queryset_from_request()
+
+
+class RegistrationSchemaResponsesDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView):
```

Comment:
We don't need this one. FE will use the top-level detail view for revision details